robyn/__init__.py

```diff
@@ -111,86 +103,64 @@ def start(self, url: str = "127.0.0.1", port: int = 8080):
-
-            process_pool = []
-            if sys.platform.startswith("win32"):
-                spawn_process(
-                    self.directories,
-                    self.request_headers,
-                    self.router.get_routes(),
-                    self.middleware_router.get_routes(),
-                    self.web_socket_router.get_routes(),
-                    self.event_handlers,
-                    socket,
-                    workers,
-                )
-
-                return process_pool
-
-            for _ in range(self.processes):
-                copied_socket = socket.try_clone()
-                process = Process(
-                    target=spawn_process,
-                    args=(
-                        self.directories,
-                        self.request_headers,
-                        self.router.get_routes(),
-                        self.middleware_router.get_routes(),
-                        self.web_socket_router.get_routes(),
-                        self.event_handlers,
-                        copied_socket,
-                        workers,
-                    ),
-                )
-                process.start()
-                process_pool.append(process)
-
-            return process_pool
+        logger.info(f"Starting server at {url}:{port}")
 
         mp.allow_connection_pickling()
 
-        if not self.dev:
-            workers = self.workers
-            socket = SocketHeld(url, port)
-
-            process_pool = init_processpool(socket)
+        if not self.config.dev:
+            run_processes(
+                url,
+                port,
+                self.directories,
+                self.request_headers,
+                self.router.get_routes(),
+                self.middleware_router.get_routes(),
+                self.web_socket_router.get_routes(),
+                self.event_handlers,
+                self.config.workers,
+                self.config.processes,
+            )
+        else:
+            event_handler = EventHandler(
+                url,
+                port,
+                self.directories,
+                self.request_headers,
+                self.router.get_routes(),
+                self.middleware_router.get_routes(),
+                self.web_socket_router.get_routes(),
+                self.event_handlers,
+                self.config.workers,
+                self.config.processes,
+            )
+            event_handler.start_server()
+            logger.info(
+                f"Dev server initialized with the directory_path : {self.directory_path}",
+                Colors.BLUE,
+            )
 
             def terminating_signal_handler(_sig, _frame):
-                logger.info(
-                    f"{Colors.BOLD}{Colors.OKGREEN} Terminating server!! {Colors.ENDC}"
-                )
-                for process in process_pool:
-                    process.kill()
+                logger.info("Terminating server!!", bold=True)
```

Comment:
@AntoineRR , can we color this? Maybe RED or GREEN?

Reply:
I did add a default green color for the `info` logs -> https://github.com/sansyrox/robyn/pull/388/files#diff-3693ef677c10179dc284219c3a15705c9b3cf7ab51c0393173879d43d1c1b2adR61

Reply:
Perfect. Thanks! :D